app/code/Magento/CatalogGraphQl/Model/Resolver/Products/DataProvider/Product/CollectionProcessor/ExtensibleEntityProcessor.php

```diff
@@ -33,16 +34,13 @@ public function __construct(JoinProcessorInterface $joinProcessor)
     }
 
     /**
-     * @param Collection $collection
-     * @param SearchCriteriaInterface $searchCriteria
-     * @param array $attributeNames
-     * @return Collection
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     * @inheritdoc
      */
     public function process(
         Collection $collection,
         SearchCriteriaInterface $searchCriteria,
-        array $attributeNames
+        array $attributeNames,
+        ContextInterface $context
```

Comment:
I think it's best to do here attributes and context optional, because not all implementations use it
also code becomes very complicated to pass all this around then not use it
Plus adding suppress unused params later..it all points optional is better